bee-api/bee-rest-api/src/endpoints/routes/api/v2/milestone_by_milestone_id.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::IpAddr;
+
+use bee_message::payload::{milestone::MilestoneId, MilestonePayload};
+use bee_runtime::resource::ResourceHandle;
+use bee_storage::access::Fetch;
+use log::error;
+use warp::{filters::BoxedFilter, reject, Filter, Rejection, Reply};
+
+use crate::{
+    endpoints::{
+        config::ROUTE_MILESTONE, filters::with_storage, path_params::milestone_id, permission::has_permission,
+        rejection::CustomRejection, storage::StorageBackend,
+    },
+    types::responses::MilestoneResponse,
+};
+
+fn path() -> impl Filter<Extract = (MilestoneId,), Error = Rejection> + Clone {
+    super::path()
+        .and(warp::path("milestones"))
```

Comment:
Route should be `/api/v2/milestones/milestoneId/{}`?

Reply:
according to the swagger spec the path for this endpoint only doesn't change, but the other milestone endpoint paths change